snippets/rust/rust.json

```diff
@@ -69,6 +69,11 @@
         "body": ["concat_idents!(${1})"],
         "description": "concat_idents!(…)"
     },
+    "dbg": {
+        "prefix": "dbg",
```

Comment:
Isn't this something that rust-analyzer already will complete?

Reply:
It does, I'll remove it as well.